### PR TITLE
Add blog post about the 1st year of Kubewarden

### PR DESCRIPTION
**Pull request by flavio**
Introduce blob post about the 1st year of Kubewarden

### CODE REVIEW

**Comment by flavio** [comment]
cc @robertsirc - @nunix  - @kubewarden/kubewarden-developers

**Review by ereslibre — APPROVED** [review]
Amazing. Love it.

**Comment by flavio** [comment]
I've added a picture too, just to show off my mad designer skills!

image

File: content/blog/2021/12/first-year-of-kubewarden.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+Finally, I'll also talk about what we plan to do during the next one.
+
+## 2021 Highlights
+
+### Project Announcement
+
+The Kubewarden project has been introduced to the massed for the 1st time during KubeCon Europe.
+Rafael and I focused explained what lead us to rethink
+how Kubernetes policies could be written and distributed.
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md:20]: It couldn't have been possible without your help and the help of all the @kubewarden/kubewarden-developers ! :pray:

**Review by kkaempf — CHANGES_REQUESTED** [review]
Great article ! Just a few wording issues. ;-)

File: content/blog/2021/12/first-year-of-kubewarden.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+During the remaining part of the year, Kubewarden talks featured at other conferences like KubeCon EU Security Day, Open Source Summit, Container Days, KubeCon NA Wasm Day, Kubernetes Community Days Italy and other live streaming events. What a year! 🤩
+
+
+### The `kwctl` Utility
+
+Shortly after KubeCon EU, we expanded the Kubewarden toolkit by releasing the [`kwctl`](https://github.com/kubewarden/kwctl/)
+utility.
+
+This is a command line tool aimed both at policy authors and Kubernetes operators.
+You can picture `kwctl` as a sort of `docker` for Kubewarden policies.
```
Comment (robertsirc) [inline content/blog/2021/12/first-year-of-kubewarden.md:32]: Can we change what we are comparing it to? I conceptualized `kwctl` closer to `kubectl`.

File: content/blog/2021/12/first-year-of-kubewarden.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+Finally, I'll also talk about what we plan to do during the next one.
+
+{{< figure class="center" src="/images/year-progress.png" alt="Year Progress Bar">}}
+
+## 2021 Highlights
+
+### Project Announcement
+
+The Kubewarden project has been introduced to the masses for the 1st time during KubeCon Europe 2021.
+Rafael and I focused explained what lead us to rethink
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md:19]: got it, fixed

File: content/blog/2021/12/first-year-of-kubewarden.md [hunk 4]
```diff
@@ -0,0 +1,122 @@
...
+In case you missed, [this blog post](/blog/2021/09/towards-a-universal-policy-platform/) gives a detailed overview about this feature.
+
+### Observability
+
+One of the major topics we tackled during the last year has been observability.
+
+We worked hard to provide a better observability story to our users: both policy authors and operators.
+This culminated with a tight integration between Kubewarden and the [OpenTelemetry project](https://opentelemetry.io/).
+
+By leveraging OpenTelemetry, policy behavior can be analyzed using modern tracing techniques. Then trace events can then be collected and inspected using tools like [Jaeger](https://www.jaegertracing.io/).
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md:56]: done

File: content/blog/2021/12/first-year-of-kubewarden.md [hunk 5]
```diff
@@ -0,0 +1,122 @@
...
+## 2021 Highlights
+
+### Project Announcement
+
+The Kubewarden project has been introduced to the masses for the 1st time during KubeCon Europe 2021.
+Rafael and I focused explained what lead us to rethink
+how Kubernetes policies could be written and distributed.
+The talk recording can be found [here](http://www.youtube.com/watch?v=Bhw5Qi78jj8), watch it to better understand Kubewarden's mission.
+
+During the remaining part of the year, Kubewarden talks featured at other conferences like KubeCon EU Security Day, Open Source Summit, Container Days, KubeCon NA Wasm Day, Kubernetes Community Days Italy and other live streaming events. What a year! 🤩
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md:23]: Done. I realized we used the same title in multiple places, but the contents of the presentations changed. Should I put a disclaimer about that?

File: content/blog/2021/12/first-year-of-kubewarden.md [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+### Observability
+
+One of the major topics we tackled during the last year has been observability.
+
+We worked hard to provide a better observability story to our users: both policy authors and operators.
+This culminated with a tight integration between Kubewarden and the [OpenTelemetry project](https://opentelemetry.io/).
+
+By leveraging OpenTelemetry, policy behavior can be analyzed using modern tracing techniques. Then trace events can then be collected and inspected using tools like [Jaeger](https://www.jaegertracing.io/).
+
+Moreover, different metrics about policies and the whole Kubewarden stack have been exposed to Prometheus. This allows a tight integration with the monitoring ecosystem that most Kubernetes Operator already use.
```
[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md]: added link

File: content/blog/2021/12/first-year-of-kubewarden.md (hunk 2 above)
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md:32]: Done

**Review by jvanz — APPROVED** [review]

File: content/blog/2021/12/first-year-of-kubewarden.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jvanz) [reply content/blog/2021/12/first-year-of-kubewarden.md]: I do not think this is a problem. As you are mentioning the conferences and not the presentations themselves, I do not think you need to explain that.

[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md]: All right. I'll go ahead and publish it

[the remark this answers is not in the document]
Comment (flavio) [reply content/blog/2021/12/first-year-of-kubewarden.md]: Thanks!

**Pull request merged**